hyperspy/io_plugins/jeol.py

```diff
@@ -306,9 +306,18 @@ def check_multiple(factor, number, string):
 
         data = np.zeros(data_shape, dtype=SI_dtype)
         datacube_reader = readcube if sum_frames else readcube_frames
-        data = datacube_reader(rawdata, data, rebin_energy, channel_number,
+        data, swept = datacube_reader(rawdata, data, sweep,
+                                      rebin_energy, channel_number,
                                width_norm, height_norm, np.iinfo(SI_dtype).max)
+        if not sum_frames and sweep == swept:
+            data = data[:sweep]  # dispose additional frame
+        if sweep < swept:
+            if sum_frames:
+                _logger.info("Broken frame %d was ignored" % (swept))
```

Comment:
Maybe to be more clear, something along the line of:
```suggestion
                _logger.info("The last frame (sweep) is incomplete because the acquisition stopped during this frame. The partially acquired frame is ignored. Use `sum_frames` to read all frames individually, including the last partially completed frame.))
```

Reply:
"The last frame (sweep) is incomplete because the acquisition stopped during this frame. The partially acquired frame is ignored. Use `sum_frames = False, read_incomplete_frame = True` to read all frames individually, including the last partially completed frame."